src/_ert/forward_model_runner/reporting/file.py

```diff
@@ -207,7 +211,7 @@ def _dump_error_file(job, error_msg):
     def _dump_ok_file():
         with open(OK_file, "w", encoding="utf-8") as f:
             f.write(
-                f"All jobs complete {time.strftime(TIME_FORMAT, time.localtime())} \n"
+                f"All forward model steps complete {time.strftime(TIME_FORMAT, time.localtime())}\n"
```

Comment:
This does change the format of the output files :thinking:

Reply:
I can take it out of this PR and maybe spare it for later when even `jobs.json` is renamed.

Reply:
so done.